aspnetcore/test/integration-tests.md

```diff
@@ -196,6 +196,45 @@ Because another test in the `IndexPageTests` class performs an operation that de
 
 [!code-csharp[](integration-tests/samples/3.x/IntegrationTestsSample/tests/RazorPagesProject.Tests/IntegrationTests/IndexPageTests.cs?name=snippet3)]
 
+## Test app with minimal hosting model
```

Comment:
I think this might go better as part of the "Basic tests with the default WebApplicationFactory" section. The basic point we need to convey is that `Program` can be used instead of `Startup` for WebApplicationFactory's generic parameter. Now in .NET 6 that we use `WebApplication.CreateBuilder()` in all of the ASP.NET Core templates, we should update the main sample to use `Program`.

We will need to point out as this PR does that you need to make `Program` visible to the test project either with InternalsVisibleTo or by making `Program` public with a partial class declaration.